src/main/java/com/treasuredata/client/TDClientApi.java

```diff
@@ -377,4 +378,8 @@
     long lookupConnection(String name);
 
     Optional<TDTableDistribution> tableDistribution(String databaseName, String tableName);
+
+    TDImportResult importFile(String database, String table, String format, File file);
```

Comment:
To integrate with Fluency, we should be able to pass byte arrays (with offset and len) cc: @komamitsu